packages/client/src/pages/forum-topic/forum-topic.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { FC, MouseEventHandler, useState } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import cn from 'classnames';
+import Button from '@components/ui/button/button';
+import style from './forum-topic.module.scss';
+import MockServer from '@/mocks/mock-server';
+import { dateFormat, FormatType } from '@/helpers/dateformat';
+import TextArea from '@/components/ui/textarea/textarea';
+import userSceleton from '@/assets/images/user-sceleton.png';
+
+const ForumTopic: FC = () => {
+    type FormFields = {
+        topicMsg: HTMLInputElement;
+    };
+
+    const { page = 1, topicId } = useParams();
+    const [newMsgModal, setNewTopicModal] = useState(false);
+
+    const newMsgModalClasses = cn(style.newMsgModal, {
+        [style.active]: newMsgModal,
+    });
+
+    const handleSubmit: React.FormEventHandler<HTMLFormElement & FormFields> = event => {
+        event.preventDefault();
+        const form = event.currentTarget;
+        const { topicMsg } = form;
+        console.log('Отправили данные формы', topicMsg.value);
+        setNewTopicModal(false);
+        topicMsg.value = '';
+    };
+
+    const handleNewMsg: MouseEventHandler<HTMLButtonElement> = event => {
+        event.preventDefault();
+        setNewTopicModal(true);
+    };
+    const handleCloseModal: MouseEventHandler<HTMLButtonElement> = event => {
+        event.preventDefault();
+        setNewTopicModal(false);
+    };
+
+    const server = new MockServer();
+    const topicData = server.getTopic(+topicId!);
+
+    const items = topicData.slice((+page! - 1) * 10, +page! * 10);
+    const isShowPrev = page && +page > 1 ? true : undefined;
+
+    const lastPage = Math.ceil(topicData.length / 10);
+    const isShowNext = page && +page < lastPage ? true : undefined;
```

Comment:
Мне кажется всё это можно упростить
```suggestion
    const isShowNext = page && +page < lastPage;
```

Reply:
Как оказалось, да) Спасибо